Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -5027,12 +5027,12 @@ script:
       - Yes
       - No
     - defaultValue: any
-      description: A comma-separated list of application object names for the rule. to create.
+      description: A comma-separated list of application object names for the rule to create.
       isArray: true
       name: application
-    - defaultValue: any
-      description: The source user for the rule to create.
+    - description: A comma-separated list of source users for the rule to create.
```

Comment:
According to the yml - did we change it in more than one command?

Reply:
yes, its the same command - one for the pan-os mode and one for the Panorama mode of the integration.

Reply:
Yes, but worth changing it to test multiple source users, or alternatively add a unit test for that logic.

Reply:
@ShacharKidor Added a UT.